generate_cscc.R

```diff
@@ -350,22 +407,32 @@ for (.rcp in rcps){
                             by = c("year",c("model_id"))]
       res_wscc = merge(res_wscc,sum_res_scc,
                        by = c("year",c("model_id")))
-      
+
       # Extrapolation SCC (before discounting)
       extrapolate_scc <- function(SD){
         if (project_val == "horizon2100") {
           .scc = 0
           .gdprate_cc_avg = 0
         } 
+        res_wscc[year == 2100,gdpcap_cc]
         if (project_val == "constant") {
           .scc = SD[year == 2100,scc]
           .gdpr = (SD[year == 2100,gdpcap_cc]/SD[year == 2100,gdpcap_cc_impulse_year])^(1/(2100 - impulse_year)) - 1
+      # I get a NaN at gdpcap_cc_impulse_year = 1087.387 but cannot find the value in the res_scc so is weird
```

Comment:
I do not know why I got this error and my thing is not really a solution but it did allow me to calculate the CSCC